templates/zerver/developer-community.html

```diff
@@ -0,0 +1,35 @@
+{% extends "zerver/portico.html" %}
+{% set entrypoint = "landing-page" %}
+
+{% set OPEN_GRAPH_TITLE = 'Modern chat for open source' %}
+{% set OPEN_GRAPH_DESCRIPTION = 'No message limits, rich moderation features, Markdown support, and a conversation model that scales to thousands of users.' %}
+
+{% block title %}
+<title>Zulip: the best group chat for open source projects</title>
```

Comment:
This title isn't right should be "The Zulip developer community".

Reply:
This is the title for the browser tab, which is same on every page, right?

Reply:
Well, it's also used for SEO / titles in Google search, which is why we want to set it appropriately for each page.